dev

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
 
+# kill any other meteor instance
+meteor_pids=$(ps ax | grep node | grep meteor | grep -v atom | awk '{print $1}')
+if [[ ! -z $meteor_pids ]]; then
```

Comment:
Maybe it's extremely likely that someone's running another meteor process on their box, but killing all other node processes on the machine could be a nasty surprise to many devs.